ui/app/src/components/BrowseFilesDialog/BrowseFilesDialogUI.tsx

```diff
@@ -98,7 +97,7 @@ export function BrowseFilesDialogUI(props: BrowseFilesDialogUIProps) {
                       item={item}
                       selected={multiSelect ? selectedArray : null}
                       onSelect={multiSelect ? onCheckboxChecked : null}
-                      onPreview={() => onPreviewImage(item)}
+                      onPreview={() => onCardSelected(item)}
```

Comment:
Leave the prop on the _UI_ but don't supply from the _Container_ dialog and default to onCardSelected.
```ts
const { ... onCardSelected, onPreviewImage = onCardSelected } = props
```